gst-libav.yaml

```
@@ -0,0 +1,87 @@
+package:
+  name: gst-libav
+  version: "1.26.0"
+  epoch: 0
+  description: GStreamer plugin for using the encoders, decoders, muxers, and demuxers provided by FFmpeg.
+  copyright:
+    - license: LGPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ffmpeg-dev
+      - gst-plugins-base-dev
+      - gstreamer-dev
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GStreamer/gstreamer
+      tag: ${{package.version}}
+      expected-commit: d31ce8e5e1aacf8f5e5beabb5c81ce2e4da5c202
+
+  - working-directory: subprojects/gst-libav
+    uses: meson/configure
+    with:
+      opts: |
+        -Db_lto=true
+      output-dir: output
+
+  - working-directory: subprojects/gst-libav
+    uses: meson/compile
+
+  - working-directory: subprojects/gst-libav
+    uses: meson/install
```

```suggestion
    pipeline:
      - uses: meson/configure
        with:
          opts: |
            -Db_lto=true
          output-dir: output
      - uses: meson/compile
      - uses: meson/install
```
we can consider avoiding repetition around usage of `working-directory`

---

It loses context if we don't use working-directory.
`ERROR: Current directory is not a meson build directory: `/home/build/output`.`

---

following is passing on workstation for me. 

<details><summary>Details</summary>
<p>

```yaml
package:
  name: gst-libav
  version: "1.26.0"
  epoch: 0
  description: GStreamer plugin for using the encoders, decoders, muxers, and demuxers provided by FFmpeg.
  copyright:
    - license: LGPL-2.0-or-later

environment:
  contents:
    packages:
      - build-base
      - busybox
      - ffmpeg-dev
      - gst-plugins-base-dev
      - gstreamer-dev
      - python3

pipeline:
  - uses: git-checkout
    with:
      repository: https://github.com/GStreamer/gstreamer
      tag: ${{package.version}}
      expected-commit: d31ce8e5e1aacf8f5e5beabb5c81ce2e4da5c202

  - working-directory: subprojects/gst-libav
    pipeline:
      - uses: meson/configure
        with:
          opts: |
            -Db_lto=true
          output-dir: output
      - uses: meson/compile
      - uses: meson/install

  - uses: strip

update:
  enabled: true
  github:
    identifier: GStreamer/gstreamer
    use-tag: true
    tag-filter-prefix: "1."

test:
  environment:
    contents:
      packages:
        - git
  pipeline:
    - uses: test/tw/ldd-check
    - name: Clone GStreamer repo and run tests
      runs: |
        git clone --branch ${{package.version}} --depth 1 https://github.com/GStreamer/gstreamer.git
        TEST_DIR=/home/build/gstreamer/subprojects/gst-libav/tests/files
        cd $TEST_DIR

        # 1. Basic plugin check
        gst-inspect-1.0 libav
        if [ $? -ne 0 ]; then
          echo "ERROR: libav plugin is missing!"
          exit 1
        fi

        # 2. Fetch APE filename
        APE_FILE=$(ls *.ape)
        if [ -z "$APE_FILE" ]; then
          echo "ERROR: No APE file found in test directory!"
          exit 1
        fi

        # 3. APE demuxing test
        gst-launch-1.0 filesrc location="$APE_FILE" ! avdemux_ape ! fakesink
        if [ $? -ne 0 ]; then
          echo "ERROR: APE demuxing failed!"
          exit 1
        fi

        # 6. FFmpeg version check
        gst-inspect-1.0 libav | grep -i ffmpeg
        if [ $? -ne 0 ]; then
          echo "ERROR: FFmpeg version check failed!"
          exit 1
        fi
```

</p>
Is melange version up to date (thinking) 
</details>